common/utils/src/main/resources/error/error-conditions.json

```
@@ -2022,8 +2022,20 @@
   },
   "INTERVAL_ARITHMETIC_OVERFLOW" : {
     "message" : [
-      "<message>.<alternative>"
+      "Integer overflow while operating with intervals."
     ],
+    "subClass" : {
+      "WITHOUT_SUGGESTION" : {
+        "message" : [
+          "Try devising appropriate values for the interval parameters."
+        ]
+      },
+      "WITH_TRY_SUGGESTION" : {
```

let's follow the existing convention in `error-conditions.json`, and rename it to
```suggestion
      "WITH_SUGGESTION" : {
```

---

`TRY` here stands for `try_X` functions which returns NULL in case of error.
So I thing actually we're following naming conventions, but it's not clear.
@mihailom-db, do you have some good points here?

---

Yeah, what @MaxGekk said makes sense, let's not complicate with saying that it is a TRY suggestion. In future it might change to some other useful suggestion.